tests/test-matrix_AMD.json

```diff
@@ -0,0 +1,22 @@
+{
+    "ubuntu22.04": {
+        "components": ["check_impi_2021", "check_rccl", "check_aocl", "check_aocc", "check_docker"],
```

Comment:
Ideally we'd have checks here to check that the ROCM drivers are installed correctly. I assume we don't because the install is happening on A100s?

Reply:
another reason is to know exactly what it means to verify rocm is correctly installed. I guess we can try checking the folder path, etc. which doesn't need to be on AMD platform

Reply:
I assume that there are some checks similar to what we do with CUDA that we could minimally do and add to later (install path check, an executable similar to `nvidia-smi` which will return version, modules showing in `lsmod`)

Reply:
added some check that I think should work on non-amd VM as well